doc/api/perf_hooks.md

```diff
@@ -98,6 +98,43 @@ added: v8.5.0
 Returns the current high resolution millisecond timestamp, where 0 represents
 the start of the current `node` process.
 
+### performance.startEventLoopTimer([options])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `resolution` {number} The sampling rate in milliseconds. Defaults to `10`.
+  * `reset` {boolean} `true` to reset the sampling data. Defaults to `false`.
```

Comment:
If this is called with `reset: true`, is `resolution` being reset to `10`? I don't think it should, but in that case we should actually document this behavior.

Reply:
No, `reset` does not change the behavior of `resolution`, but it's also important to know that `resolution` is set on every call to start the timer. The value is not persisted and reused between calls. So if I do...

```
startEventLoopTimer({ resolution: 15 })
stopEventLoopTimer()
startEventLoopTimer()
stopEventLoopTimer()
```

Then the second call to `startEventLoopTimer()` will use the default resolution of 10.

Reply:
Specifically, all `reset: true` does is zero out the actual counters.

Reply:
Consider this example: 

```js
startEventLoopTimer({ resolution: 15 })
stopEventLoopTimer()
startEventLoopTimer({ reset: true }) // what is the value of resolution here?
stopEventLoopTimer()
```

Reply:
10, because that's the default when it's not passed.